Code/Support/RestKit-Prefix.pch

```diff
@@ -5,6 +5,11 @@
 #ifdef __OBJC__
     #import <Foundation/Foundation.h>
 
+// Fix for compilation on iOS 6 where this is not defined.
+#if !defined NS_DESIGNATED_INITIALIZER
```

Comment:
You sent a patch that fixes a real problem, but part of submitting pull requests is getting feedback. As the maintainer of RestKit, it would be irresponsible to merge any patch that was sent regardless of code quality and formatting. Over time, that would destroy the codebase. If a polite request to match the project formatting in your patch upsets you that much, please find other projects to contribute to instead.